File: Acupay/Audit/Entity/AuditTrail.vb
Imports System
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace Auditing

    <Table("audittrail")>
    Public Class AuditTrail

        <Key>
        Public Property RowID As Integer?

        Public Property Created As Date

        Public Property CreatedBy As Integer?

        Public Property OrganizationID As Integer?

        Public Property ViewID As Integer?

        Public Property FieldChanged As String

        Public Property ChangedRowID As Integer?

        Public Property OldValue As String

        Public Property NewValue As String

        Public Property ActionPerformed As String

    End Class

End Namespace

File: AccuPay.Core/Entities/To Be Deleted/AuditTrail.cs
/*
 Delete this once AuditTrail Table is finally deleted.
 */

using System.ComponentModel.DataAnnotations.Schema;

namespace AccuPay.Core.Entities
{
    [Table("audittrail")]
    public class AuditTrail : BaseEntity
    {
        public int? OrganizationID { get; set; }
    }
}
